data/src/main/java/com/globant/data/database/entity/PokemonSupertypeRoom.kt

```diff
@@ -0,0 +1,10 @@
+package com.globant.data.database.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "pokemon_supertype_table")
+data class PokemonSupertypeRoom(
```

Comment:
this class should be PokemonSupertypeEntity, if you change the database you should rename everything

Reply:
refactored, thanks